src/components/shared/header.cr

```diff
@@ -53,7 +53,7 @@ class Shared::Header < BaseComponent
   end
 
   private def nav_links
-    nav_link("Guides", Guides::GettingStarted::Installing.path)
+    nav_link("Learn", Learn::Index.path)
```

Comment:
It may be redundant, but we could also have both links... 🤔

Reply:
I do like having a direct links to docs. Learn is great for "this is my first time" vs docs being good for anyone of experience getting to reference quickly. Does it look bad with both? If not, I'd suggest that. If so, we could a drop down or put "Learn" prominently on the homepage instead? The idea being that the nav is for people looking for something particular, but the homepage is more general information if you're trying to understand what lucky is.

Reply:
Yeah, I really like having both in the top navbar as well.

Reply:
Added the Guides link back for now!